infrastructure/terraform/modules/browser/dns.tf

```
@@ -0,0 +1,47 @@
+
+resource "cloudflare_zone" "dns" {
+    zone = "${var.domain}"
+}
+
+resource "cloudflare_record" "static" {
+  zone_id = "${cloudflare_zone.dns.id}"
+  name    = local.static_domain
+  value   = "${aws_cloudfront_distribution.s3_distribution.domain_name}"
+  type    = "CNAME"
+  ttl     = 1
+}
+
+resource "cloudflare_record" "star" {
```

I think wildcard is a better name

```suggestion
resource "cloudflare_record" "wildcard" {
```